cmd/statusd/main.go

```diff
@@ -48,7 +48,7 @@ var (
 
 	// stats
 	statsEnabled = flag.Bool("stats", false, "Expose node stats via /debug/vars expvar endpoint or Prometheus (log by default)")
-	statsAddr    = flag.String("stats.addr", "127.0.0.1:8080", "HTTP address with /debug/vars endpoint")
+	statsAddr    = flag.String("stats.addr", "0.0.0.0:8080", "HTTP address with /debug/vars endpoint")
```

Comment:
Another option would be to leave `127.0.0.1` and set it in docker-compose in `wnode-test-cluster/` with `-stats-addr=0.0.0.0:8080` but I think this is fine as well.